pkg/services/cloudmigration/cloudmigration.go

```diff
@@ -5,18 +5,18 @@ import (
 )
 
 type Service interface {
+	// tokens
```

Comment:
These comments don't seem to add a lot of context 🤔

Reply:
you're right! removed https://github.com/grafana/grafana/pull/86406/commits/73a59576921aad48e1d47582bd9dfaed51950680